src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -181,13 +185,10 @@ function getNewFileHandle() {
 
 #### Save changes to the original file
 
-To write data to disk, I needed to create a [`FileSystemWriter`][fs-writer]
-by calling `createWriter()` on the file handle, then call `write()` to do
-the write. If permission to write hasn’t been granted already, the browser
-will prompt the user for permission to write to the file first (during
-`createWriter()`). The `write()` method takes a string, which is what we
-want for a text editor, but it can also take a [BufferSource][buffersource],
-or a [Blob][blob].
+You can find all the code for saving changes to a file in my [text
```

Comment:
We need to keep the permission prompting stuff in and as close to the same wording as originally written. It was content requested by the sec/priv team.

Reply:
Unless I missed something, the restructuring retains all the information previously in the doc, so it sounds good to me modulo one additional sentence I want to keep in for emphasis (commented on above).